src/js/components/Overview/charts/BentoPie.js

```diff
@@ -120,7 +121,7 @@ function BentoPie({ data, height }) {
         <path d={`M${sx},${sy}L${mx},${my}L${ex},${ey}`} stroke={fill} fill="none" />
         <circle cx={ex} cy={ey} r={2} fill={fill} stroke="none" />
         <text x={ex + (cos >= 0 ? 1 : -1) * 12} y={ey + 3} textAnchor={textAnchor} style={currentTextStyle}>
-          {labelShortName(t(name))}
+          {labelShortName(name)}
```

Comment:
for my own understanding, why is this now untranslated?

Reply:
It is translated earlier so no need after

Reply:
maybe put a comment mentioning where this comes from/where it gets translated to make it easy to understand in the future then?